### PR TITLE
Updates notes on exoscale

### PR DESCRIPTION
**Pull request by glasnt**
Some additional formatting changes, cleaning up of page.

### CODE REVIEW

File: how-to/interact-storage.rst [hunk 1]
```diff
@@ -230,45 +248,6 @@ Secret Access/Secret Key  secret from DSN value
 ------------------------  ---------------------------------------------------
 ========================  ====================  =============================
 
-s3cmd
```
Comment (evildmp) [inline how-to/interact-storage.rst]: Do you suggest we not mention s3cmd at all?

File: how-to/interact-storage.rst [hunk 2]
```diff
@@ -99,7 +98,6 @@ See the `AWS S3 regions table
 <http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region>`_ for more
 information about regions and their names.
 
-
```
Comment (evildmp) [inline how-to/interact-storage.rst]: Keep two lines before a new heading.

File: how-to/interact-storage.rst (hunk 1 above)
Comment (glasnt) [reply how-to/interact-storage.rst]: given awscli will work in all regions, it's not required.

**Comment by glasnt** [comment]
@evildmp :shipit: ❓

**Pull request merged**